src/reformat_libyuv.c

```diff
@@ -93,9 +93,9 @@ static int avifABGRToJ420(const uint8_t * src_abgr,
     // A temporary buffer is needed to swap the R and B channels before calling ARGBToJ420().
     uint8_t * src_argb;
     const int src_stride_argb = width * 4;
-    const uint64_t soft_allocation_limit = 16384; // Arbitrarily chosen trade-off between CPU and memory footprints.
+    const int soft_allocation_limit = 16384; // Arbitrarily chosen trade-off between CPU and memory footprints.
     int num_allocated_rows;
-    if ((height == 1) || ((uint64_t)src_stride_argb * height <= soft_allocation_limit)) {
+    if ((height == 1) || ((uint64_t)src_stride_argb * height <= (uint64_t)soft_allocation_limit)) {
```

Comment:
Optional: We could also cast `src_stride_argb` to `int64_t` instead. Then we should not need to cast `soft_allocation_limit` because there won't be a signed-unsigned comparison.

If you make this change, make a similar change to the cast in line 102.